lib/paypal_express/api.rb

```diff
@@ -301,6 +301,29 @@ def get_payer_info(token,
         response
       end
 
+      def create_billing_aggrement(token,
+                                   kb_account_id,
+                                   kb_tenant_id,
+                                   payment_processor_account_id,
+                                   kb_payment_id = nil,
```

Comment:
You need not assign default value as `nil` . If the param is not passed, it'll be `nil`.

Reply:
you need to define the default value otherwise function call will not happen if the parameter is not passed

Reply:
Alright.